orchestrator/config.py

```
@@ -56,8 +56,27 @@ class Config:
     ANTHROPIC_API_KEY: str = os.getenv("ANTHROPIC_API_KEY")
-    LLM_PROVIDER: str = os.getenv("LLM_PROVIDER", "openai")
-    LLM_MODEL: str = os.getenv("LLM_MODEL", "gpt-4")
+    
+    # LLM settings - loaded from database system_settings (fallback to env vars)
+    @property
+    def LLM_PROVIDER(self) -> str:
+        """Get LLM provider from system settings (database) or environment"""
+        try:
+            from services.llm_provider.manager import get_system_setting
+            return get_system_setting("orchestrator_llm", "provider", os.getenv("LLM_PROVIDER", "openai"))
+        except:
```

Ruff (0.14.4)

Do not use bare `except`

(E722)